lib/record_store/record.rb

```diff
@@ -17,7 +17,8 @@ def escape(value)
       end
 
       def quote(value)
-        %("#{escape(value)}")
+        result = Record.needs_long_quotes?(value) ? escape(value).scan(/.{1,255}/).join('" "') : escape(value)
```

Comment:
Can't we use directly `needs_long_quotes?` without specifying the class given we're already in the same place?